sources/api/migration/migrations/v1.19.0/add-additional-ecs-settings/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "add-additional-ecs-settings"
+version = "0.1.0"
+authors = ["Arnaldo Garcia Rincon <agarrcia@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+publish = false
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
```

Comment:
We don't need to keep copying this in migration `Cargo.toml`s since there's no readme.

Reply:
I'll fix this in my follow up PR.